src/plugins/inputs/away.js

```diff
@@ -0,0 +1,19 @@
+"use strict";
+
+exports.commands = ["away", "back"];
+
+exports.input = function(network, chan, cmd, args) {
+	if (cmd === "away") {
+		let reason = " ";
+
+		if (args.length > 0) {
```

Comment:
@YaManicKill, did you mean this because when user types `/away`, the command that gets sent is `network.irc.raw("AWAY", " ");` and therefore user is marked as away even with no reason given?

I am actually happy with having a way to get away without giving a specific reason, but I definitely see your point. IMO, we have 2 options:

- Current option: `/away <message>` that does what it says, marks you as away, with an optional reason. `/back` that sets you "unaway".
- Close-to-the-protocol option: `/away message` sets you away with a reason, `/away` sets you back.

I have looked at other popular IRC clients and both options exist. That being said, most IRC clients out there have a terrible UX and I think The Lounge setting a higher and more modern bar on that end is a really good thing.
Overall, I am OK with both options, but I tend to like @xPaw's as it's simpler to use (and it's not entirely unexpected either, while having to type `/away` to set oneself back is rather counter-intuitive). I don't care to not reproduce the protocol 1:1 because it's a client, not a library.